oak_functions/examples/metrics/module/.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[build]
+target-dir = "/workspace/oak_functions/target/wasm"
+out-dir = "/workspace/oak_functions/bin"
```

Comment:
Maybe those end up being the default?

Reply:
So I was not sure why we set `target-dir` for the examples and not simply write in the default directory. Do you know? 

I am also not 100% sure why we need `out-dir`.

Reply:
I think it was an attempt to know upfront where the target files end up, since otherwise that depends on whether the crate is in a workspace, etc. But I think now we can actually parse the actual target folder via cargo metadata, so perhaps both target dir and out dir are not necessary any more? Basically, if things work without them (i.e. all tests and examples pass), then it's fine to remove them, they serve no other purpose (that I remember).

Reply:
Definitely!

Reply:
#2472